app/posts/2020-01-03-set-up-a-design-history.md

```diff
@@ -41,7 +41,7 @@ npm install
 Before uploading your design history to a public URL, it can be useful to preview it locally, on your own computer. To do this type:
 
 ``` text
-npm start
+npm run dev
 ```
```

Comment:
Prototype kit users will be familiar with using `npm start` to run something. I like the design history's symmetry with that at the moment.